precompiles/ics20/types.go

```diff
@@ -85,7 +85,7 @@ func NewTransferAuthorization(method *abi.Method, args []interface{}) (common.Ad
 	return grantee, transferAuthz, nil
 }
 
-// NewMsgTransfer returns a new transfer message from the given arguments.
+// CreateMsgTransfer returns a new transfer message from the given arguments.
 func NewMsgTransfer(method *abi.Method, args []interface{}) (*transfertypes.MsgTransfer, common.Address, error) {
```

Comment:
Usually the comment starts with the name of the function, either change the comment or the function name
```suggestion
// CreateMsgTransfer returns a new transfer message from the given arguments.
func CreateMsgTransfer(method *abi.Method, args []interface{}) (*transfertypes.MsgTransfer, common.Address, error) {
```

Reply:
My bad, I updated the name of the function without updating the comment. Thanks @GAtom22 !